docs/Data_Portal/Users_Guide/quick_start.md

```diff
@@ -16,6 +16,8 @@ On the upper-left is the GDC Data Portal logo, which links to the home page of t
 * **Repository**: a shortcut to the [Repository](repository.md) tool in the Analysis Center. The Repository tool allows exploration of files associated with a cohort.
 
 On the right are the following features:
+* **Video Guides**: links to videos demonstrating the various features of the Data Portal.
```

Comment:
Same comment above regarding periods at the end of single-sentence bullet-points